samples/cpp/stable_diffusion/README.md

```diff
@@ -0,0 +1,46 @@
+# Stable Diffusion C++ Image Generation Pipeline
+
+This example showcases inference of text to image models like Stable Diffusion 1.x, 2.x, LCM. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample features `ov::genai::Text2ImagePipeline` and uses a text prompt as input source.
+
+Users can change the sample code and play with the following generation parameters:
+
+- Change width or height of generated image
+- Generate multiple images per prompt
+- Adjust a number of inference steps
+- Play with [guidance scale](https://huggingface.co/spaces/stabilityai/stable-diffusion/discussions/9) (read [more details](https://arxiv.org/abs/2207.12598))
+- (SD 1.x, 2.x only) Add negative prompt when guidance scale > 1
+
+## Download and convert the models and tokenizers
+
+The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
+
+It's not required to install [../../requirements.txt](../../requirements.txt) for deployment if the model has already been exported.
+
+```sh
+pip install --upgrade-strategy eager -r ../../requirements.txt
+optimum-cli export openvino --model dreamlike-art/dreamlike-anime-1.0 --task stable-diffusion --weight-format fp16 dreamlike_anime_1_0_ov/FP16`
+```
+
+## Run
+
+`stable_diffusion ./dreamlike_anime_1_0_ov/FP16 'cyberpunk cityscape like Tokyo New York  with tall buildings at dusk golden hour cinematic lighting'`
+
+### Examples
+
+Prompt: `cyberpunk cityscape like Tokyo New York  with tall buildings at dusk golden hour cinematic lighting`
+
+   ![](./512x512.bmp)
+
+## Supported models
+
+Models can be downloaded from [OpenAI HiggingFace](https://huggingface.co/openai). This sample can run the following list of models, but not limitied to:
+
+- [botp/stable-diffusion-v1-5](https://huggingface.co/botp/stable-diffusion-v1-5)
+- [dreamlike-art/dreamlike-anime-1.0](https://huggingface.co/dreamlike-art/dreamlike-anime-1.0)
+- [SimianLuo/LCM_Dreamshaper_v7](https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7)
+
+## Troubleshooting
+
+- Image generated with HuggingFace / Optimum Intel is not the same generated by this C++ sample:
+
+C++ random generation with MT19937 results differ from `numpy.random.randn()` and `diffusers.utils.randn_tensor`. So, it's expected that image generated by Python and C++ versions provide different images, because latent images are initialize differently. Users can implement its own random generator derived from `ov::genai::Generator` and pass it to `Text2ImagePipeline::generate` method.
```

Comment:
```suggestion
C++ random generation with MT19937 results differ from `numpy.random.randn()` and `diffusers.utils.randn_tensor`. So, it's expected that image generated by Python and C++ versions provide different images, because latent images are initialize differently. Users can implement their own random generator derived from `ov::genai::Generator` and pass it to `Text2ImagePipeline::generate` method.
```

Reply:
good finding.